specification/communication/Communication.Messages/tspconfig.yaml

```diff
@@ -58,3 +58,13 @@ options:
       name: "@azure-rest/communication-messages"
       description: "Azure client library for Azure Communication Messages services"
     flavor: azure
+  "@azure-tools/typespec-go":
+    service-dir: "sdk/resourcemanager/communication"
+    package-dir: "armcommunication"
+    module: "github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}"
+    fix-const-stuttering: true
+    flavor: "azure"
+    generate-samples: true
+    generate-fakes: true
+    head-as-boolean: true
+    inject-spans: true
```

Comment:
```suggestion
```
you do not need to add config for go since the spec is for data plane. you could add suppression to bypass the validation error.